components/service/nimbus/src/main/java/mozilla/components/service/nimbus/ui/NimbusBranchItemViewHolder.kt

```diff
@@ -20,12 +19,15 @@ class NimbusBranchItemViewHolder(
     private val nimbusBranchesDelegate: NimbusBranchesAdapterDelegate,
     private val selectedIconView: ImageView,
     private val titleView: TextView,
-    private val summaryView: TextView,
-    private val selectedBranch: String
+    private val summaryView: TextView
 ) : RecyclerView.ViewHolder(view) {
 
-    internal fun bind(branch: Branch) {
-        selectedIconView.isVisible = selectedBranch == branch.slug
+    internal fun bind(branch: Branch, selectedBranch: String) {
+        selectedIconView.visibility = if (selectedBranch == branch.slug) {
+            View.VISIBLE
+        } else {
+            View.GONE
+        }
```

Comment:
`isVisible` does the same thing as `visibility`. I would prefer to keep it as `isVisible` due to the simplicity of it without having to specify `View.VISIBLE` and `View.GONE`.

Reply:
I suspect this is also what is causing the tests to fail as well.